homework5/task_4.py

```diff
@@ -0,0 +1,16 @@
+
+from math import ceil
+
+n = int(input("enter a number: "))
+if 0 < n < 50:
+    for i in range(1, n+1):
+        if i==1:
+            print ((" "*(n-i-1)), "/*\\")
+        elif i>1 and i< n:
+            print(" "*(n-i),"/","*"*(2*i-1),"\\",sep="")
+        elif i <= n:
+            print(" "*ceil(n-2),"| |")
+    
+
+else:
+    print("enter a valid number")
```

Comment:
მგონი აქ ოდნავ აცდენაა ამოცანის პირობასთან და გთავაზობ ალტერნატივას:
n = int(input('Enter a natural number below 50: '))

if not 0 < n < 50:
    print('Entered number does not meet the requirement.')
    exit(1)

for i in range(1, n):
    print(' ', end = '')
print('*')

for i in range(1, n):
    for j in range(1, n + i + 1):
        if j < n - i:
            print(' ', end='')
        elif j < n:
            print('/', end='')
        elif j > n:
            print('\\', end='')
        else:
            print('|', end='')
    print()

for i in range(1, n):
    print(' ', end = '')
print('|')